docs/document/website/versioned_docs/version-1.4.0/guides/version-conflicts-issue-solution.md

```diff
@@ -0,0 +1,61 @@
+---
+id: version-1.4.0-version-conflicts-issue-solution
+title: Version Conflicts Issue and Solution
+sidebar_label: Version Conflicts Issue and Solution
+original_id: version-conflicts-issue-solution
+---
+## Overview
+
+Version conflicts have been a problem for KUC users before the release of v1.4.0. This guide will help users understand the version conflicts issue and how to adapt and upgrade KUC to the latest version starting from v1.4.0.
+
+## The Problem of Versioning
```

Comment:
I think if we choose to not use "the" then a better alternative would be "Problem with versioning". What do you think?

Reply:
Or should we use "Versioning issue"? What do you of think of it, @TomokoMiyake ?

Reply:
@huuchi1778 
As communicated, let's go with "Problems of versioning". (Use plural for it seems good according to the investigation in case we don't add "the" in front)